tests/topotests/lib/common_config.py

```diff
@@ -722,12 +724,12 @@ def start_topology(tgen):
                 os.mkdir("{}".format(rname))
                 os.system("chmod -R go+rw {}".format(rname))
                 os.chdir("{}/{}".format(TMPDIR, rname))
-                os.system("touch zebra.conf bgpd.conf")
+                os.system("touch zebra.conf bgpd.conf pimd.conf")
```

Comment:
I'm seeing bgp being started for these new tests.  Why are we starting bgp?  And conversely do the bgp tests now start pim?

Reply:
I have done changes accordingly . Daemons will be started as per feature test, so here BGPd will not be started